ros/src/styx/bridge.py

```diff
@@ -52,12 +55,19 @@ def __init__(self, conf, server):
         '/final_waypoints': self.callback_path
         }
 
+        self.image_queue = Queue.Queue(1)
+
         self.subscribers = [rospy.Subscriber(e.topic, TYPE[e.type], self.callbacks[e.topic])
                             for e in conf.subscribers]
 
         self.publishers = {e.name: rospy.Publisher(e.topic, TYPE[e.type], queue_size=1)
                            for e in conf.publishers}
 
+        self.publish_camera_rate = 1  # HZ
```

Comment:
1 Hz seems quite low I think.

Reply:
For my poor CPU it barely manageable :(

However, perhaps we can commit a larger value (5? 10?) and I can tweak it on my system as needed.